filebeat/docs/reference/configuration/reload-configuration.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[filebeat-configuration-reloading]]
+=== Reload Configuration
+
+experimental[]
+
+Reload configuration allows to dynamically reload prospector configuration files. A glob can be defined which should be watched
+ for prospector configuration changes. New prospectors will started / stopped accordingly. This is especially useful in
+ container environments where 1 container is used to tail logs from services in other containers on the same host.
+
+The configuration in the main filebeat.yml config file looks as following:
+
+[source,yaml]
+------------------------------------------------------------------------------
+filebeat.reload.modules:
```

Comment:
It is actually `reload.prospectors` not modules. Copy / paste error from metricbeat ...